owa/app/js/components/view_log_entries/viewLogEntries.jsx

```diff
@@ -9,127 +9,308 @@
-}
 
-const AdvancedSearch = () => (
-    <div className="advanced_wrapper">
-        <form>
+    else{this.setState({
+      searchFilters: Object.assign({}, this.state.searchFilters, {[name]: value})
+    });
+    }
+  }
+
+  handleChangeStart(date) {
+    this.setState({startDate: date
+    });
+  }
 
-            <br/>
-            <fieldset>
-                <div className="col-sm-6 col-md-4">
-                    <label className="search_lbl" name="source">Source Name</label>
+  handleChangeEnd(date) {
+    this.setState({endDate: date, dateChanged: !this.state.dateChanged});
+  }
+
+  componentDidMount() {
+    this.getLogEntries();
+    this.getIdentifierSources();
+  }
 
-                    <Input type="select" id="source">
-                        <option>1</option>
-                    </Input>
+  resetSearchForm() {
+    this.setState({
+      startDate:'',
+      endDate: moment(new Date()),
+      generatedByValue:'',
+      searchFilters: {
+        source: '',
+        identifier: '',
+        comment: '',
+        generatedBy: ''
+      },
+      isHidden: !this.state.isHidden,
+      dateChanged: false
+    });
+    this.getLogEntries();
+  }
 
+  render() {
+    let logEntries = this.state.logEntries
+    if (this.state.filteredLogEntries) {
+      logEntries = logEntries.filter(LogEntry => {
+        let formattedDate = moment(LogEntry.dateGenerated).format('DD/MM/YYYY')
+        return LogEntry.source.toLowerCase().includes(this.state.filteredLogEntries.toLowerCase())||
+             LogEntry.identifier.toLowerCase().includes(this.state.filteredLogEntries.toLowerCase()) || 
+             LogEntry.comment.toLowerCase().includes(this.state.filteredLogEntries.toLowerCase()) || 
+             formattedDate.includes(this.state.filteredLogEntries) || 
+             (LogEntry.generatedBy.username.toLowerCase()).includes(this.state.filteredLogEntries.toLowerCase())
+      })
+    }
+
+    return (
+      <div className="logwrapper">
+        <div className="search_label">
+          <i className=""></i>
+          <span>Search Log Entries</span>
+        </div>
+        <div className="logSearch">
+          <div className="input-group">
+            <input
+              type="text"
+              placeholder="Search"
+              className="log_searchbox"
+              defaultValue={this.state.filteredLogEntries}
+              onChange={event => this.setState({filteredLogEntries: event.target.value})}/>
+            <span className="input-group-btn">
+              <button className="btn btn-secondary" type="button">< FaSearch/></button>
+            </span>
+          </div>
+          <button
+            className="searchlink"
+            onClick={this.advancedSearchView}>More Search Options</button>
+        </div>
+        <div className="logs_table_area">
+          <div >{!this.state.isHidden && <div className="advanced_wrapper">
+            <form id="advancedSearchForm">
+              <br/>
+              <fieldset>
```

Comment:
Having 2 div tags on the same line is confusing to read, consider using indentations to separate the elements hierarchically